src/envs/playroom.py

```diff
@@ -186,17 +249,18 @@ def types(self) -> Set[Type]:
 
     @property
     def options(self) -> Set[ParameterizedOption]:
-        # To update later
-        return {self._Pick, self._Stack, self._PutOnTable}
+        return {self._Pick, self._Stack, self._PutOnTable, self._Move,
+                self._OpenDoor, self._CloseDoor, self._TurnOnDial,
+                self._TurnOffDial}
 
     @property
     def action_space(self) -> Box:
         # dimensions: [x, y, z, rotation, fingers]
         # x, y, z location for the robot's disembodied hand
-        # robot's heading is the angle (rotation * pi/2) in standard position
-        lowers = np.array([self.x_lb, self.y_lb, 0.0, -2.0, 0.0],
+        # robot's heading is the angle (rotation * pi) in standard position
```

Comment:
any particular reason why this is changing? fine by me, just curious

Reply:
i kept confusing myself because i thought it was just a factor of pi but then i'd have to remember that it was actually pi/2 :P i think this is more intuitive